src/targets/public/main.jsx

```diff
@@ -41,7 +42,21 @@ function init () {
   if (directdownload) {
     cozy.client.files.getDownloadLinkById(id).then(
       link => window.open(`${cozy.client._url}${link}`, '_SELF')
-    )
+    ).catch(e => {
+      cozy.bar.init({
+        appName: data.cozyAppName,
+        appEditor: data.cozyAppEditor,
+        iconPath: data.cozyIconPath,
+        lang: data.cozyLocale,
+        replaceTitleOnMobile: true
+      })
```

Comment:
I was mostly thinking of moving it to [l.41](https://github.com/enguerran/cozy-drive/blob/33916c81bb6066f2c2b72d5c173fcc5ba3d88ef1/src/targets/public/main.jsx#L41), when the client is initialized too — since we'll want the bar regardless of what happens?